manim/__init__.py

```diff
@@ -8,9 +10,25 @@
 __version__ = importlib_metadata.version(__name__)
 
 
+import sys
+
 # Importing the config module should be the first thing we do, since other
 # modules depend on the global config dict for initialization.
 from ._config import *
+
+# Workaround to set the renderer passed via CLI args *before* importing
+# Manim's classes (as long as the metaclass approach for switching
+# between OpenGL and cairo rendering is in place, classes depend
+# on the value of config.renderer).
+for ind, arg in enumerate(sys.argv):
```

Comment:
I'd rather `ind` get renamed to `i` (which is conventional and thus doesn't impair readability imo) or `index`

Reply:
Fine with me: 4ebe3df776320405e4102d0bc5a587fa871436fe